README.md

```diff
@@ -120,6 +120,31 @@ We use [appear.in](http://appear.in) for video call communications. We have a bl
 
 ## Engineering
 
+### Open Source
+At Caramba we embrace the open value whenever it's possible, for example, when it's not a project for a client. Some examples of good candidates for open sourcing would be: command line tools, example apps and libraries. Before deciding for open sourcing a project, it should be proposed to the team, and once decided.

```

Comment:
The last phrase looks incomplete `and once decided.` and once decided what happens? :P